inspirehep/modules/search/static/js/search/app.js

```diff
@@ -22,9 +22,6 @@
  */
 
 require([
-    'angular',
```

Comment:
Should be fine as the dependencies are defined in `settings.js` so I think they can be removed.